contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/gobblin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
This is for the so-called double shadow. It would be quite difficult to do otherwise.